server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -38,7 +41,7 @@ pub struct ApplicationIn {
     )]
     pub name: String,
 
-    #[validate(range(min = 1, message = "Application rate limits must be at least 1 if set"))]
+    #[validate(custom = "validate_rate_limit")]
```

Comment:
I honestly think it was better before, the same way you did it for name below...